proc/variables.go

```diff
@@ -1567,6 +1606,34 @@ func (scope *EvalScope) variablesByTag(tag dwarf.Tag, cfg LoadConfig) ([]*Variab
 			vars = append(vars, val)
 		}
 	}
+	if len(vars) <= 0 {
+		return vars, nil
+	}
+
+	// prefetch the whole chunk of memory relative to this variables
```

Comment:
AFAIK this is impossible. Escaping locals are replaced by `&v` entries that are stored on the stack, the compiler does not produce DW_OP_deref instructions (and we don't support them). But I'll add a check anyway, as all this could change.